test/create-run-publish-app.ps1

```diff
@@ -0,0 +1,35 @@
+[cmdletbinding(PositionalBinding = $true)]
```

Comment:
I've considered it, but haven't had the time to try it. Last time I try converting build scripts (not this repo but another one), I found that powershell on Linux was terribly buggy. That was a long time ago so it's probably worth checking out again.